test/IndexTest.cpp

```diff
@@ -527,3 +527,14 @@ TEST(IndexTest, loggingAndSettingOfParallelParsing) {
         HasSubstr("but has to be specified"));
   }
 }
+
+TEST(IndexTest, getBlankNodeManager) {
+  // The `blankNodeManager_` is initialized after initializing the Index itself.
+  // Therefore we expect a throw when the getter is called by an
+  // uninitialized Index.
+  Index index{ad_utility::makeUnlimitedAllocator<Id>()};
+  EXPECT_ANY_THROW(index.getBlankNodeManager());
+
+  const Index& index2 = getQec("")->getIndex();
+  EXPECT_NO_THROW(index2.getBlankNodeManager());
+}
```

Comment:
Also add a test for the correct reading and writing of the minimal local blank node index
just use getQec(...) with a kg that has some blank nodes, and then access the blank node manager of the returned index to check that the number of blank nodes was propagated correctly.